project-blitz/project_blitz/__init__.py

```diff
@@ -1,5 +1,5 @@
 from project_blitz import automation, resources_and_configurations
 
 
-defs = automation.defs
-# defs = resources_and_configurations.defs
+# defs = automation.defs
```

Comment:
@PedramNavid / @tacastillo how do we want to manage project definitions? Right now I just toggle between them with comments.

Reply:
we have a couple options, but i think maybe the simplest is to have one module per talk, and the definitions in that module, with no global definitions.
then you can run `dagster dev -m my_module`

Reply:
Confirmed that the following works great - thank you.
```
dagster dev -m project_blitz.automation
# or
dagster dev -m project_blitz.resources_and_configurations
```

Reply:
Added an option to choose the project when using `make run`:
```
> make run

Select the example project to run:
1) automation
2) resources-and-configuration
#? 2
```